chapter_4/chapter_4.md

```diff
@@ -0,0 +1,21 @@
+# Chapter 4:
+- General approach for building a model:
+1- Recognize the set of variables we are going to work with. The observable ones are called data, the unobservable ones, such as means or rates, are called parameters.
+2- Each variable is defined in terms of a probability distribution or in terms of other variables.
+3- The combination of variables and their probability distributions defines what it called a *joint generative model*. It can be used to simulate hypothetical observations or to analyze real ones.
+- The Gaussian is a member of a family of fundamental natural distributions known as the exponential family. All of the members of this family are important for working science, because they populate our world.
+- Using Gaussian distributions:
+    - Ontological justification: the world is full of Gaussian distributions, approximately.
+    - By the epistemological justification, the Gaussian represents a particular state of ignorance. When all we know or are willing to say about a distribution of measures (measures are continuous values on the real number line) is their mean and variance, then the Gaussian distribution arises as the most consistent with our assumptions.
```

Comment:
I would also consider calling it "the Gaussian distribution" here, and everywhere, unless you found examples in the book of it being called merely "the Gaussian"